server/webhook.go

```diff
@@ -49,9 +49,11 @@ func (wh *webhook) Events() StringSet {
 	return wh.eventTypes
 }
 
-func (wh webhook) PostToChannel(p *Plugin, instanceID types.ID, channelId, fromUserId string) (*model.Post, int, error) {
+func (wh webhook) PostToChannel(p *Plugin, instanceID types.ID, channelId, fromUserId, subscriptionName string) (*model.Post, int, error) {
 	if wh.headline == "" {
 		return nil, http.StatusBadRequest, errors.Errorf("unsupported webhook")
+	} else if p.getConfig().DisplaySubscriptionNameInNotifications && subscriptionName != "" {
+		wh.headline = fmt.Sprintf("%s\nSubscription: %s", wh.headline, subscriptionName)
```

Comment:
You can make the subscription name bold using this markdown:

```suggestion
		wh.headline = fmt.Sprintf("%s\nSubscription: **%s**", wh.headline, subscriptionName)
```